au4/src/effects/effects_base/internal/effectsprovider.cpp

```diff
@@ -182,3 +187,241 @@ muse::Ret EffectsProvider::performEffect(AudacityProject& project, Effect* effec
 
     return success ? muse::make_ok() : muse::make_ret(Ret::Code::UnknownError);
 }
+
```

Comment:
I can assume that this is an unmodified copy-paste (except for formatting), right? Hence I won't really review its content.

Reply:
Right!